### PR TITLE
Update Magento 1 Quickstart section to use OpenMage (docs only), fixes #2283

### LINKED ISSUE #2283: Docs: Replace Magento 1 (after end of support) with OpenMage (open source clone of Magento 1)
Ref: https://github.com/drud/ddev/blob/master/docs/users/cli-usage.md#magento-1-quickstart

- Link to "Magento release page" is wrong. The link goes to https://github.com/OpenMage/magento-mirror which is a mirror for magento 1 releases

- I agree, to not recommend Magento 1 (delivered by Magento an Adobe company).
- Let's remove this recommendation for Magento1 and replace it with [OpenMage/Magento-LTS](https://github.com/OpenMage/magento-lts)

### PR DESCRIPTION
**Pull request by kkrieger85**
## The Problem/Issue/Bug:

Docs related

## How this PR Solves The Problem:

Use current state of information about Magento 1 and OpenMage

## Manual Testing Instructions:

Read the text
Klick on the links
Do all steps on your system

## Related Issue Link(s):

 https://github.com/drud/ddev/issues/2283

## Release/Deployment notes:
Update Docs section about Magento1 /OpenMage quickstart

### CODE REVIEW

**Comment by rfay** [comment]
Thanks, not to worry about the failed tests, the test runners had a bad night.

File: docs/users/cli-usage.md [hunk 1]
```diff
@@ -265,18 +265,21 @@ Successfully started example-site
...
-4. `ddev config` and accept the defaults.
+1. Download OpenMage from [release page](https://github.com/OpenMage/magento-lts/releases).
+2. Make a directory for it, for example `mkdir ~/workspace/OpenMage` and change directory to the new directory.
+3. `ddev config` and accept the defaults.
+4. (Install sample data - see below)
 5. `ddev start`
 6. Follow the URL to the base site.
 
-You may want the [Magento 1 Sample Data](https://magento.com/tech-resources/download#download1759) for experimentation.
+You may want the [Magento 1 Sample Data](https://raw.githubusercontent.com/Vinai/compressed-magento-sample-data/1.9.1.0/compressed-magento-sample-data-1.9.1.0.tgz) for experimentation:
```
Comment (rfay) [inline docs/users/cli-usage.md:277]: I imagine this link won't be a permalink, so could you change it to be how to find it? I assume it's always in github.com/Vinai/Vinai/compressed-magento-sample-data ?

Comment (kkrieger85) [reply docs/users/cli-usage.md:277]: This file hasn't changed for 6 years. And I am sure, it won't change.

Maybe, somewhen OpenMage could publish their own sample data. But it's either planned nor does a issue for missing sample data exists.

**Comment by rfay** [comment]
I'd like to bring this in soon. The one remaining request was

> Please confirm that you've manually tested these instructions on a new project.

**Comment by kkrieger85** [comment]
Manually tested:

```
git clone https://github.com/OpenMage/magento-lts && cd magento-lts    
ddev config (with defaults only)
ddev start
```

=> Magento default start page is shown => works fine

**Pull request merged**